Nodejs/Product/Nodejs/Options/NodejsIntellisenseOptionsPage.cs

```diff
@@ -148,6 +169,41 @@ public NodejsIntellisenseOptionsPage()
             SaveString(CompletionCommittedBySetting, CompletionCommittedBy);
             SaveBool(SaveToDiskSetting, SaveToDisk);
 
+        }
+
+        private string _toolsVersion;
+        private string GetTypeScriptToolsVersion() {
+            if (_toolsVersion == null) {
+                _toolsVersion = string.Empty;
+                try {
+                    // Start with the Nodejs tools path to retrieve the IDE path
+                    var idePath = Assembly.GetExecutingAssembly().Location;
+                    while (!string.IsNullOrEmpty(idePath) && !StringComparer.OrdinalIgnoreCase.Equals(Path.GetFileName(idePath), "IDE")) {
+                        var parentInfo = Directory.GetParent(idePath);
+                        idePath = parentInfo != null ? parentInfo.FullName : string.Empty;
+                    }
+
+                    if (string.IsNullOrEmpty(idePath)) {
+                        return _toolsVersion;
+                    }

```

Comment:
This method won't work in the experimental hive because the assembly will be in a different location. Instead use ```VSSPROPID_InstallDirectory``` like the following to get the VS installation directory.
```cs
object installDirAsObject = null;
IVsShell shell = NodejsPackage.Instance.GetService(typeof(SVsShell)) as IVsShell;
if (shell != null) {
    shell.GetProperty((int)__VSSPROPID.VSSPROPID_InstallDirectory, out installDirAsObject);
}
idePath = CommonUtils.NormalizeDirectoryPath((string)objInstallationDir) ?? string.Empty;
```